single_tech_samples/databricks/sample2_enterprise_azure_databricks_environment/README.md

```diff
@@ -0,0 +1,273 @@
+   4. Azure Public Ip for the Firewall.
+   5. Azure Routing tables with UDR (User Defined Routes) configured to enforce regulation of traffic through the firewall.
+   6. Azure Data Lake Gen2 storage ABFS endpoint accessible only via Azure Private Link.
+   7. Azure key vault to store secrets and access tokens accessible via Azure private link.
+
+Details about [how to use this sample](#3-how-to-use-this-sample) can be found in the later sections of this document.
+
+### 1.2. Architecture
+
+The architecture of the solution is aligned with the security baselines for [Azure Databricks](https://docs.microsoft.com/en-us/azure/databricks/security/). The following diagram captures the high-level design.
+
+![alt text](../Common_Assets/Images/ADE_Architecture.png "Logo Title Text 1")
+
+In this sample, a shell script is used to orchestrate the deployment. The following diagram illustrates the deployment process flow.
+
+![alt text](../Common_Assets/Images/ADE_Deployment.png "Logo Title Text 1")
+
+#### 1.2.1. Patterns
+
+Following are the cloud design patterns being used by this sample:
+
+- [External Configuration Store pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/external-configuration-store): Configuration for the deployment is persisted externally as a parameter file separate from the deployment script.
+- [Federated Identity pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/federated-identity): Azure active directory is used as the federated identity store to enable seamless integration with enterprise identity providers.
+- [Valet Key pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/valet-key): Azure key vault is used to manage the secrets and access toked used by the services.
+- [Gatekeeper pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/gatekeeper): The Azure firewall acts as a gatekeeper for all external traffic flowing in.
+
+### 1.3. Technologies used
+
+The following technologies are used to build this sample:
+
+- [Azure Databricks](https://azure.microsoft.com/en-au/free/databricks/)
+- [Azure Storage](https://azure.microsoft.com/en-au/services/storage/data-lake-storage/)
+- [Azure Key Vault](https://azure.microsoft.com/en-au/services/key-vault/)
+- [Azure Virtual networks](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview)
+- [Azure Firewall](https://docs.microsoft.com/en-us/azure/firewall/overview)
+- [Azure Route tables](https://docs.microsoft.com/en-us/azure/virtual-network/manage-route-table)
+- [Azure Public IP](https://docs.microsoft.com/en-us/azure/virtual-network/public-ip-addresses)
+- [Azure Private Links](https://docs.microsoft.com/en-us/azure/private-link/private-link-overview)
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/)
+- [Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview)
+
+## 2. Well-Architected Framework (WAF)
```

Comment:
To make this section a bit more relevant to the sample, move the Security section to be the top and call it out that the focus of the sample is around Network Security and Data exfiltration prevention, and how the sample enables these. 

Then add "Other considerations:" then add the other points such as Cost Optimization, etc. Change these into simple bullet points instead of having its own sections, then just add sub-bullet points underneath them.

The WAF section's purpose is not meant to elaborate on **ALL** WAF concepts, but rather only elaborate on the specific WAF concept (in this case, Security) which is the focus of the sample. :)

Reply:
@namitms Addressing

Reply:
@devlace Resolved